### PR TITLE
[LMS-376] Additional Sample Data and Scripting

### PR DESCRIPTION
**Pull request by stephenfuqua**

### CODE REVIEW

**Review by lalrg — COMMENTED** [review]
I'd also add a readme file similar to what we have in https://github.com/Ed-Fi-Alliance-OSS/Engage-Online-Learners-Starter-Kit/tree/main/vm-docs to the directories for scripts, docs, and packer.
I also think the output of the script with Packer could give more feedback if possible (While downloading it does not bring an easy way of knowing what's the current progress of the download).

**Comment by stephenfuqua** [comment]
> I also think the output of the script with Packer could give more feedback if possible (While downloading it does not bring an easy way of knowing what's the current progress of the download).

That's a Packer thing, so not something I can improve.

File: scripts/Install-EdFiTechnologySuite.ps1 [hunk 1]
```diff
@@ -47,6 +37,12 @@
     * Ed-Fi ODS Admin App for Suite 3, version 2.2
     * Ed-Fi Analytics Middle Tier, latest development work (`main` branch)
     * Ed-Fi LMS Toolkit, latest development work (`main` branch).
+
+    Note: the Analytics Middle Tier and LMS Toolkit are installed from the
+    `main` branch by default because these are the primary two systems
+    that are under development to support this Starter Kit. Experimentally,
+    you can change the versions to any tag from those code repositories and
+    the install process will alternately download that tag instead of `main`.
```
[the remark this answers is not in the document]
Comment (stephenfuqua) [reply scripts/Install-EdFiTechnologySuite.ps1:45]: Yes to both - good to copy into the readme files, and very important for PowerShell scripts like this to have appropriate documentation in them. Note that you can see all of this documentation when you run `get-help Install-EdFiTechnologySuite.ps1`. I'm going to run [PSDocs](https://github.com/microsoft/PSDocs) on this to generate some Markdown documentation from the PowerShell.

[the remark this answers is not in the document]
Comment (stephenfuqua) [reply scripts/Install-EdFiTechnologySuite.ps1]: I've now provided README files in the `scripts` and `packer` directories.

**Review by lalrg — APPROVED** [review]
I run it locally with Packer and it worked fine, had the expected software installed in the VM... Code and documentation are also looking good.

**Pull request merged**